data/channels.csv

```diff
@@ -27695,6 +27699,7 @@ StarMaaMoviesHD.in,Star Maa Movies HD,,Disney Star,Disney India,IN,,Hyderabad,c/
 StarMaaMusic.in,Star Maa Music,Maa Music,Disney Star,Disney India,IN,,Hyderabad,c/IN,tel,music,FALSE,2008-01-01,,,https://www.hotstar.com/in/channels/star-maa,https://i.imgur.com/vxP9L8p.png
 StarMax.pk,Star Max,,,,PK,,,c/PK,eng,,FALSE,,,,,https://i.imgur.com/patxMdp.png
 StarMovies.in,Star Movies,,Disney Star,Disney India,IN,,Mumbai,c/IN,eng;hin;tam,movies,FALSE,1991-04-01,,,http://www.starmovies.in/,https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/STAR_Movies_logo.svg/512px-STAR_Movies_logo.svg.png
+StarMovies.pt,Star Movies,,The Walt Disney Company Portugal,The Walt Disney Company Iberia S.L.,PT,,Lisboa,c/PT,por,movies,FALSE,2024-02-07,,,https://www.startv.pt/programacao/starmovie,https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/Star_Movies_2023.svg/1920px-Star_Movies_2023.svg.png

```

Comment:
Image must be under 512px.

Reply:
https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/Star_Movies_2023.svg/512px-Star_Movies_2023.svg.png